lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -78,6 +80,15 @@ class ArtworkRail extends React.Component {
     if (url) { SwitchBoard.presentNavigationViewController(this, url) }
   }
 
+  geneQueryLink(rail) {
+    if (!rail.context) { return '' }
+    // Pull out any params, removing the first (which is `__data_id__`)
+    // and turn that into a query
+    return rail.context && rail.context.href + '?' +
+      Object.keys(rail.params).slice(1).map(function(key) {
```

Comment:
Can you make the omission of the `__data_id__` key more robust, rather than relying on it to be at a certain index? Maybe `Object.keys(_.omit(rail.params, '__data_id__'))` ?